client/game/component/Slider.jsx

```diff
@@ -60,24 +59,16 @@ export default ({
           }}
         />
         {aiPrediction && (
-          <div
-            className="prediction ai"
-            style={{ left: `calc(${aiPercentage}% - 6.5px)` }}
-          >
+          <div className="prediction ai" style={{ left: `calc(${aiPercentage}% - 6.5px)` }}>
             <div className="prediction-content">
               <div className="prediction-point" />
               <div className="prediction-line" />
-              <div className="prediction-box">
-                AI Prediction – {Math.round(aiPercentage)}%
-              </div>
+              <div className="prediction-box">AI Prediction – {Math.round(aiPercentage)}%</div>
```

Comment:
You need to start using prettier (https://github.com/prettier/prettier-vscode). You can run it on save, so we all have the same output. Or you are using it, as you might be given that these look like automatic reformats, you should use the default configuration (maybe you're changing the max line width?)

Reply:
@npaton Sorry my bad, i accidentally change the max line width. will use the default configuration onward.